README.md

```diff
@@ -4,14 +4,13 @@ The source files for https://bevyengine.org. This includes official Bevy news an
 
 ## Zola
 
-The Bevy website is built using the Zola static site engine. In our experience, it is fast, flexible, and straightforward to use. [Check it out here](https://www.getzola.org/)!
+The Bevy website is built using the Zola static site engine. In our experience, it is fast, flexible, and straightforward to use.
 
 To check out any local changes you've made:
-1. Download Zola.
-2. Clone the Bevy Website git repo and move to that directory:
-  1. `git clone https://github.com/bevyengine/bevy-website.git`
-  2. `cd bevy-website`
-3. Start the Zola server with `zola serve`.
+- [Download Zola](https://www.getzola.org/).
```

Comment:
In addition of fixing the nested list, I removed the "Check it out" link and added it to the list itself.